data_structures/linked_list/doubly_linked_list.py

```diff
@@ -33,6 +33,24 @@ def __str__(self):
             current = current.next
         return "<-->".join(str(node) for node in nodes)
```

Comment:
I find the "<—>" notation to be a lot of visual clutter.  Why not just add an `.__iter__()` method and then `return str(list(self))`?

Reply:
Hi sir,
I have added in a linked list iterator class for iterating over the linked list
and removed the <--> representation.

However, I have left the __str__(self) same and just removed the <--> from the string as
this would keep visualizing the linked list if someone wants to visualize the data in the Linked list while debugging as opposed to the memory address that it spits out

I had previously used the  <--> representation as I thought it would make the links between nodes more clear


@cclauss I have made the changes